Makefile.Common

```diff
@@ -55,8 +61,10 @@ install-tools: $(TOOLS_BIN_NAMES)
 $(TOOLS_BIN_DIR):
 	mkdir -p $@
 
+$(TOOLS_MOD_DIR)/go.mod:
```

Comment:
Without this empty rule I'm getting the error below, so telling make to not try to build it

```
mkdir -p /c/Users/pjanotti/src/otel/collector/contrib/.tools
make: *** No rule to make target '/c/Users/pjanotti/src/otel/collector/contrib/internal/tools/go.mod', needed by '/c/Users/pjanotti/src/otel/collector/contrib/.tools/genqlient'.  Stop.
```